diagnostics.md

```diff
@@ -166,6 +166,17 @@ This test depends on the container engine being healthy (see [check_container_en
 Depending on what part of this check failed, there are various fixes and workarounds. Most however will involve a
 restrictive local network, or an unreliable connection.
 
+
+### check_image_corruption
+#### Summary
+This check tests that container images on the device can be read.
+
+#### Triage
+Check that the device has sufficient disk space (see [check_localdisk](#check_localdisk)).  Container images may need to be purged (see [balena-engine image prune](https://github.com/balena-os/balena-engine-cli/blob/master/docs/reference/commandline/image_prune.md)]).
```

Comment:
If this check fails, we can conclude that at least one image has been corrupted somehow. Additionally there _could_ be underlying storage corruption, though that's not dispositive. I'm not sure `image prune` would actually fix this issue, since the supervisor does that regularly and these images will likely be associated with running containers.

Reply:
In that case, is there any thing the user *can* do here?  Or is this a case where we're saying "Keep an eye on this; if it keeps happening you'll need to think about replacing the device/upgrading the storage/run it through the dishwasher"?

Reply:
Since the issue may just be layer corruption caused by the engine, there are workarounds that support agents will often try. We don't advertise/share those methods since they could potentially cause issues, but we recommend that users reach out for us to perform the cleanup.